### ISSUE TITLE
[Drop] Droprates are ridiculously high

### ISSUE THREAD

**Issue by Cheesedip** [Stormforge-gg/Bugtracker-Fordring#1695]
**Description**: 
Drop rates are way too high

**Current behaviour**:
Too many green, blue items

**Expected behaviour**: 
Preferably return to similar droprates of previous core

**Comment by Cold-Martijn** [comment]
Any specific dungeons/mobs?

**Comment by Cheesedip** [comment]
I've done DKT hc a few time today and I've seen the same blues drop maybe 5 times. If it helps I will write down the names of the items and the mobs and post them under this ticket. 
I've also heard and seen for myself in HoL e.g. the item librarian's paper cutter dropping many times. 

Also Frostweave bandage manual drops maybe 3-4 times in any heroic.

**Comment by Cold-Martijn** [comment]
> 

Yes, it definitly does help to know the items and the mobs that drop them :)

**Comment by Cheesedip** [comment]
- In Ahn'Kahet I've seen 2 and 3 greens drop off the same NPC: bonegrinder and frostbringer
- Manual: heavy frostweave bandage, as mentioned this drops too much, about 3 times on average per hc I'd say.
- Also I've seen Recipe: elixer of empowerment drop a lot, but could be coincidence
- Waistguard of the risen knight - DKT hc
- Helm of lightning halls - HOL hc
- Buring skull pendant - Gundrak hc - Drakkari Earthshaker
- Dragon Prow Amulet - Oculus hc
- Forstbridge Orb - Oculus hc
- Runecaster's Mantle, Tattered Castle drape - Utgarde Keep hc

**Comment by Vishiswaz** [comment]
Yeah the global loot chance for boe greens and blues are way too high, as it is on most private servers except Atlantiss ones.

**Comment by Kripp0** [comment]
World boe is fine,slightly better than the previous core.Its mostly dungeon drops.

**Comment by Vishiswaz** [comment]
It's more than slightly.

**Comment by Kripp0** [comment]
Mojo Masked Crusher is not BoE

**Comment by Cheesedip** [comment]
> Mojo Masked Crusher is not BoE

Thanks, removed from list
